code/obj/item/storage/small_storage_parent.dm

```diff
@@ -366,30 +124,26 @@
 	item_state = "gun"
 	flags = FPRINT | EXTRADELAY | TABLEPASS | CONDUCT
 	w_class = 4.0
-	max_wclass = 3
 
 	New()
 		..()
 		src.setItemSpecial(null)
+		AddComponent(/datum/component/storage, can_hold = list(/obj/item/reagent_containers/pill), max_wclass = 3)
```

Comment:
Is there a reason for limiting the rock-it to pills?

Reply:
An oversight from copying and pasting!